src/foraging_gui/Foraging.py

```diff
@@ -3934,6 +3978,16 @@ def _Start(self):
             # stop lick interval calculation
             self.GeneratedTrials.lick_interval_time.stop()  # stop lick interval calculation
 
+            # validate behavior session model and document validation errors if any
+            try:
+                AindBehaviorSessionModel(**self.behavior_session_model.model_dump())
+            except ValidationError as e:
+                logging.error(str(e), extra={'tags': [self.warning_log_tag]})
```

Comment:
@bruno-f-cruz is this way a good way to validate the model? I thought the model wouldn't allow attributes to be set to variables with incorrect types after initialization, but after some testing, it doesn't seem to be the case?